tools/upp4.1-mpas/parm/postxconfig-NT.txt
1
84
MPASPRS
0
ncep_emc
v2003
local_tab_yes1
fcst
oper
fcst
fcst
hour
nws_ncep
gfs_avn
complex_packing_spatial_diff
2nd_ord_sptdiff
fltng_pnt
lossless
12
HGT_ON_ISOBARIC_SFC
?
1
tmpl4_0
HGT
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
13
TMP_ON_ISOBARIC_SFC
?
1
tmpl4_0
TMP
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
15
DPT_ON_ISOBARIC_SFC
?
1
tmpl4_0
DPT
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
16
SPFH_ON_ISOBARIC_SFC
?
1
tmpl4_0
SPFH
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
17
RH_ON_ISOBARIC_SFC
?
1
tmpl4_0
RH
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
18
UGRD_ON_ISOBARIC_SFC
?
1
tmpl4_0
UGRD
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
19
VGRD_ON_ISOBARIC_SFC
?
1
tmpl4_0
VGRD
?
?
isobaric_sfc
0
?
47
100. 200. 300. 500. 700. 1000. 2000. 3000. 5000. 7000. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
20
VVEL_ON_ISOBARIC_SFC
?
1
tmpl4_0
VVEL
?
?
isobaric_sfc
0
?
37
10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
5.0
0
0
0
?
?
?
21
ABSV_ON_ISOBARIC_SFC
?
1
tmpl4_0
ABSV
?
?
isobaric_sfc
0
?
4
25000. 50000. 70000. 85000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
23
MSLET_ON_MEAN_SEA_LVL
?
1
tmpl4_0
MSLET
NCEP
?
mean_sea_lvl
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
105
PRES_ON_MEAN_SEA_LVL
?
1
tmpl4_0
PRMSL
?
?
mean_sea_lvl
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
138
PRES_ON_SPEC_HGT_LVL_ABOVE_GRND_2m
?
1
tmpl4_0
PRES
?
?
spec_hgt_lvl_above_grnd
0
?
1
2.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
106
TMP_ON_SPEC_HGT_LVL_ABOVE_GRND_2m
?
1
tmpl4_0
TMP
?
?
spec_hgt_lvl_above_grnd
0
?
1
2.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
112
SPFH_ON_SPEC_HGT_LVL_ABOVE_GRND_2m
?
1
tmpl4_0
SPFH
?
?
spec_hgt_lvl_above_grnd
0
?
1
2.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
113
DPT_ON_SPEC_HGT_LVL_ABOVE_GRND_2m
?
1
tmpl4_0
DPT
?
?
spec_hgt_lvl_above_grnd
0
?
1
2.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
114
RH_ON_SPEC_HGT_LVL_ABOVE_GRND_2m
?
1
tmpl4_0
RH
?
?
spec_hgt_lvl_above_grnd
0
?
1
2.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
64
UGRD_ON_SPEC_HGT_LVL_ABOVE_GRND_10m
?
1
tmpl4_0
UGRD
?
?
spec_hgt_lvl_above_grnd
0
?
1
10.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
65
VGRD_ON_SPEC_HGT_LVL_ABOVE_GRND_10m
?
1
tmpl4_0
VGRD
?
?
spec_hgt_lvl_above_grnd
0
?
1
10.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
24
PRES_ON_SURFACE
?
1
tmpl4_0
PRES
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
25
HGT_ON_SURFACE
?
1
tmpl4_0
HGT
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
26
TMP_ON_SURFACE
?
1
tmpl4_0
TMP
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
116
TSOIL_ON_DEPTH_BEL_LAND_SFC
?
1
tmpl4_0
TSOIL
?
?
depth_bel_land_sfc
4
2 2 2 2
4
0. 10. 40. 100.
depth_bel_land_sfc
4
2 2 2 2
4
10. 40. 100. 200.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
117
SOILW_ON_DEPTH_BEL_LAND_SFC
?
1
tmpl4_0
SOILW
NCEP
?
depth_bel_land_sfc
4
2 2 2 2
4
0. 10. 40. 100.
depth_bel_land_sfc
4
2 2 2 2
4
10. 40. 100. 200.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
225
SOILL_ON_DEPTH_BEL_LAND_SFC
?
1
tmpl4_0
SOILL
NCEP
?
depth_bel_land_sfc
4
2 2 2 2
4
0. 10. 40. 100.
depth_bel_land_sfc
4
2 2 2 2
4
10. 40. 100. 200.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
36
SOILM_ON_DEPTH_BEL_LAND_SFC
?
1
tmpl4_0
SOILM
?
?
depth_bel_land_sfc
1
2
1
0.
depth_bel_land_sfc
1
2
1
200.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
170
VEG_ON_SURFACE
?
1
tmpl4_0
VEG
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
218
VGTYP_ON_SURFACE
?
1
tmpl4_0
VGTYP
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
219
SOTYP_ON_SURFACE
?
1
tmpl4_0
SOTYP
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
30
LFTX_ON_ISOBARIC_SFC_500-1000hpa
?
1
tmpl4_0
LFTX
NCEP
?
isobaric_sfc
0
?
1
50000.
isobaric_sfc
0
?
1
100000.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
31
4LFTX_ON_SPEC_PRES_ABOVE_GRND
?
1
tmpl4_0
4LFTX
NCEP
?
spec_pres_above_grnd
0
?
1
18000.
spec_pres_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
75
PLI_ON_SPEC_PRES_ABOVE_GRND
?
1
tmpl4_0
PLI
?
?
spec_pres_above_grnd
0
?
1
3000.
spec_pres_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
32
CAPE_ON_SURFACE
?
1
tmpl4_0
CAPE
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
566
BEST_CAPE_ON_SPEC_PRES_ABOVE_GRND
?
1
tmpl4_0
CAPE
?
?
spec_pres_above_grnd
0
?
1
18000.
spec_pres_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
584
UNSTABLE_CAPE_ON_SPEC_PRES_ABOVE_GRND
?
1
tmpl4_0
CAPE
?
?
spec_pres_above_grnd
0
?
1
25500.
spec_pres_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
107
CIN_ON_SURFACE
?
1
tmpl4_0
CIN
?
?
surface
0
?
1
0.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
567
BEST_CIN_ON_SPEC_PRES_ABOVE_GRND
?
1
tmpl4_0
CIN
?
?
spec_pres_above_grnd
0
?
1
18000.
spec_pres_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
585
UNSTABLE_CIN_ON_SPEC_PRES_ABOVE_GRND
?
1
tmpl4_0
CIN
?
?
spec_pres_above_grnd
0
?
1
25500.
spec_pres_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
80
PWAT_ON_ENTIRE_ATMOS_SINGLE_LYR
?
1
tmpl4_0
PWAT
?
?
entire_atmos_single_lyr
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
162
HLCY_ON_SPEC_HGT_LVL_ABOVE_GRND
?
1
tmpl4_0
HLCY
?
?
spec_hgt_lvl_above_grnd
0
?
1
3000.
spec_hgt_lvl_above_grnd
0
?
1
0.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
87
ACM_APCP_ON_SURFACE
?
1
tmpl4_8
APCP
?
ACM
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
33
ACM_ACPCP_ON_SURFACE
?
1
tmpl4_8
ACPCP
?
ACM
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
34
ACM_NCPCP_ON_SURFACE
?
1
tmpl4_8
NCPCP
?
ACM
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
151
WTMP_ON_SURFACE
?
1
tmpl4_0
WTMP
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
5.0
0
0
0
?
?
?
37
LCDC_ON_LOW_CLOUD_LYR
?
1
tmpl4_0
LCDC
?
?
low_cloud_lyr
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
38
MCDC_ON_MID_CLOUD_LYR
?
1
tmpl4_0
MCDC
?
?
mid_cloud_lyr
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
39
HCDC_ON_HIGH_CLOUD_LYR
?
1
tmpl4_0
HCDC
?
?
high_cloud_lyr
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
188
PRES_ON_CONVECTIVE_CLOUD_BOT_LVL
?
1
tmpl4_0
PRES
?
?
convective_cloud_bot_lvl
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
189
PRES_ON_CONVECTIVE_CLOUD_TOP_LVL
?
1
tmpl4_0
PRES
?
?
convective_cloud_top_lvl
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
143
AVE_CDCON_ON_ENTIRE_ATMOS
?
1
tmpl4_8
CDCON
?
AVE
entire_atmos
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
173
PRES_ON_MAX_WIND
?
1
tmpl4_0
PRES
?
?
max_wind
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
174
HGT_ON_MAX_WIND
?
1
tmpl4_0
HGT
?
?
max_wind
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
175
UGRD_ON_MAX_WIND
?
1
tmpl4_0
UGRD
?
?
max_wind
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
176
VGRD_ON_MAX_WIND
?
1
tmpl4_0
VGRD
?
?
max_wind
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
221
HPBL_ON_SURFACE
?
1
tmpl4_0
HPBL
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
252
REFC_ON_ENTIRE_ATMOS
?
1
tmpl4_0
REFC
NCEP
?
entire_atmos
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
180
VIS_ON_SURFACE
?
1
tmpl4_0
VIS
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
142
INST_ULWRF_ON_SURFACE
?
1
tmpl4_0
ULWRF
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
275
BRTMP_ON_TOP_OF_ATMOS
?
1
tmpl4_0
BRTMP
?
?
top_of_atmos
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
156
INST_DSWRF_ON_SURFACE
?
1
tmpl4_0
DSWRF
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
44
SFCR_ON_SURFACE
?
1
tmpl4_0
SFCR
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
45
FRICV_ON_SURFACE
?
1
tmpl4_0
FRICV
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
48
NLAT_ON_SURFACE
?
1
tmpl4_0
NLAT
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
49
ELON_ON_SURFACE
?
1
tmpl4_0
ELON
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
50
LAND_ON_SURFACE
?
1
tmpl4_0
LAND
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
1.0
0
0
0
?
?
?
266
AVE_ALBDO_ON_SURFACE
?
1
tmpl4_8
ALBDO
?
AVE
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
54
PRES_ON_TROPOPAUSE
?
1
tmpl4_0
PRES
?
?
tropopause
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
177
HGT_ON_TROPOPAUSE
?
1
tmpl4_0
HGT
?
?
tropopause
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
55
TMP_ON_TROPOPAUSE
?
1
tmpl4_0
TMP
?
?
tropopause
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
56
UGRD_ON_TROPOPAUSE
?
1
tmpl4_0
UGRD
?
?
tropopause
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
57
VGRD_ON_TROPOPAUSE
?
1
tmpl4_0
VGRD
?
?
tropopause
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
62
HGT_ON_0C_ISOTHERM
?
1
tmpl4_0
HGT
?
?
0C_isotherm
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
421
MAX_REF_ON_SPEC_HGT_LVL_ABOVE_GRND_1km
maximum Updraft Helicity on Specified Height Level Above Ground
1
tmpl4_8
MAXREF
NCEP
MAX
spec_hgt_lvl_above_grnd
0
?
1
1000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
-3.0
0
0
0
?
?
?
425
AVE_DZDT_ON_SIGMA_LVL_0.5-0.8
average Vertical velocity between sigma lvl 0.5 and 0.8
1
tmpl4_8
DZDT
?
AVE
sigma_lvl
1
2
1
50.
sigma_lvl
1
2
1
80.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
-4.0
0
0
0
?
?
?
430
VUCSH_ON_SPEC_HGT_LVL_ABOVE_GRND_0-1km
Vertical u-component shear between 0 to 1000m Above Ground
1
tmpl4_0
VUCSH
?
?
spec_hgt_lvl_above_grnd
0
?
1
0.
spec_hgt_lvl_above_grnd
0
?
1
1000.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
-4.0
0
0
0
?
?
?
431
VVCSH_ON_SPEC_HGT_LVL_ABOVE_GRND_0-1km
Vertical v-component shear between 0 to 1000m Above Ground
1
tmpl4_0
VVCSH
?
?
spec_hgt_lvl_above_grnd
0
?
1
0.
spec_hgt_lvl_above_grnd
0
?
1
1000.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
-4.0
0
0
0
?
?
?
432
VUCSH_ON_SPEC_HGT_LVL_ABOVE_GRND_0-6km
Vertical u-component shear between 0 to 6000m Above Ground
1
tmpl4_0
VUCSH
?
?
spec_hgt_lvl_above_grnd
0
?
1
0.
spec_hgt_lvl_above_grnd
0
?
1
6000.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
-4.0
0
0
0
?
?
?
433
VVCSH_ON_SPEC_HGT_LVL_ABOVE_GRND_0-6km
Vertical v-component shear between 0 to 6000m Above Ground
1
tmpl4_0
VVCSH
?
?
spec_hgt_lvl_above_grnd
0
?
1
0.
spec_hgt_lvl_above_grnd
0
?
1
6000.
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
-4.0
0
0
0
?
?
?
245
GUST_ON_SURFACE
?
1
tmpl4_0
GUST
?
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
274
INST_ULWRF_ON_TOP_OF_ATMOS
?
1
tmpl4_0
ULWRF
NCEP
?
top_of_atmos
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
161
INST_TCDC_ON_ENTIRE_ATMOS
?
1
tmpl4_0
TCDC
?
?
entire_atmos
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
3.0
0
0
0
?
?
?
258
TPFI_ON_ISOBARIC_SFC
?
1
tmpl4_0
TPFI
?
?
isobaric_sfc
0
?
1
30000.
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
4.0
0
0
0
?
?
?
260
HGT_ON_CLOUD_CEILING
?
1
tmpl4_0
HGT
?
?
cloud_ceilng
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
6.0
0
0
0
?
?
?
444
LTNG_ON_SURFACE
lightning
1
tmpl4_0
LTNG
NCEP
?
surface
0
?
0
?
?
0
?
0
?
?
?
0
0.0
0
0.0
?
0
0.0
0
0.0
1
1.0
0
0
0
?
?
?
118
CNWAT_ON_SURFACE
?
1
tmpl4_0
CNWAT

tools/upp4.1-mpas/sorc/ncep_post.fd.new/params.F
module params_mod

!- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       implicit none
!
! file: params.f
! defines physical constants and smoothing coefficients
! last update: 09/06/2006
!
!     SMOOTHING PARAMETERS.
    integer, parameter :: KSMUD=4
    integer, parameter :: KSLPD=1
    integer, parameter :: KSSLP=2
    integer, parameter :: KSFI=2
    integer, parameter :: KST=2
    integer, parameter :: KSUV=2
    integer, parameter :: KSRH=2
    integer, parameter :: KSAV=2
    integer, parameter :: KSLI=2
    integer, parameter :: KSLP=2
    integer, parameter :: NSUMD=1
                                                                                   
    real, parameter :: SMTHA=-1.8E-4
    real, parameter :: SMTHB=16.0
!
! CONSTANTS.
    real, parameter :: CM1=2937.4
    real, parameter :: CM2=4.9283
    real, parameter :: CM3=23.5518
    real, parameter :: D00=0.0
    real, parameter :: D1=0.1
    real, parameter :: D01=0.01
    real, parameter :: D001=0.001
    real, parameter :: D50=0.50
    real, parameter :: H999=999.
    real, parameter :: H1=1.0
    real, parameter :: H2=2.0
    real, parameter :: H4=4.0
    real, parameter :: H99999=99999.
    real, parameter :: H1M12=1.E-12
    real, parameter :: H1000=1000.
    real, parameter :: H1M5=1.E-5
    real, parameter :: D125=.125
    real, parameter :: D25=0.25
    real, parameter :: H100=100.
    real, parameter :: H10000=10000.
    real, parameter :: H10E5=1.E5
    real, parameter :: CAPA=0.28589641E0
    real, parameter :: D115=0.115
    real, parameter :: D608=0.608
    real, parameter :: D05=0.05
    real, parameter :: D0065=0.0065
    real, parameter :: H1E2=100.
    real, parameter :: H1E4=1.E4
    real, parameter :: H1M2=1.E-2
    real, parameter :: G=9.81
    real, parameter :: RD=287.04
    real, parameter :: ROG=RD/G
    real, parameter :: GI=1./G
    real, parameter :: OVERRC=1.10
    real, parameter :: AD05=OVERRC*D05
    real, parameter :: CFT0=OVERRC-H1
    real, parameter :: PI=3.141592653589793
    real, parameter :: DTR=3.1415926/180.
    real, parameter :: RTD=1./DTR
    real, parameter :: ERAD=6.370E6
    real, parameter :: CP=1004.6
    real, parameter :: RCAPA=1./CAPA
    real, parameter :: P1000=1000.E2,DP10M=110
    real, parameter :: ELWV=2.50E6
    real, parameter :: CPDR=CP/RD
    real, parameter :: RG=1./GI
    real, parameter :: ELDR=ELWV/RD
    real, parameter :: BETA=.00367
    real, parameter :: BTG=BETA*G
    real, parameter :: FMW=18.015
    real, parameter :: FMD=28.964
    real, parameter :: EPS=FMW/FMD
    real, parameter :: ONEPS=1.-EPS
    real, parameter :: TFRZ=273.15
    real, parameter :: RDOCP=RD/CP
    real, parameter :: LHEAT=2.5E6
    real, parameter :: LHTOCP=LHEAT/CP
    real, parameter :: EPSQ2=0.02
    real, parameter :: PQ0=379.90516
    real, parameter :: A2=17.2693882
    real, parameter :: A3=273.16
    real, parameter :: A4=35.86
    real, parameter :: EPSQ=1.E-12
#ifdef GSDRAPR
    real, parameter :: QCLDmin=1.E-6  ! was 1.E-6
#else
    real, parameter :: QCLDmin=1.E-5  ! was 1.E-6
#endif
    real, parameter :: CLFRmin=D1
    real, parameter :: NLImin=1.E3
! move definition of NLImax and T_ICE to MICROINIT 2012012018
!    real, parameter :: NLImax=5.E3
!    real, parameter :: T_ICE=-30.
!    real, parameter :: TRAD_ice=0.5*T_ICE+TFRZ
    real, parameter :: Qconv=0.1E-3
!
    real, parameter :: STBOL=1./5.67E-8
    real, parameter :: DBZmin=-20.
    real, parameter :: abscoef=8.0
    real, parameter :: abscoefi=5.0
    real, parameter :: XLAI=4.0         ! Leaf area index
    real, parameter :: RHmin=1.0E-6     ! minimal RH bound
    real, parameter :: SMALL=1.E-6
!    real, parameter :: PTHRESH=0.000000
    real, parameter :: ELIVW=2.72E6
    real, parameter :: ELOCP=ELIVW/CP
!
    real, parameter :: GAMMA=6.5E-3
    real, parameter :: RGAMOG=RD*GAMMA/G

end module params_mod
